### PR TITLE
curator-operator-leader-election-roles-rb

### PR DESCRIPTION
**Pull request by skanthed**

### CODE REVIEW

File: curator-operator-system/base/rbac.authorization.k8s.io/rolebindings/rolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: leader-election-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: leader-election-role
+subjects:
+- kind: ServiceAccount
```
Comment (dystewart) [inline curator-operator-system/base/rbac.authorization.k8s.io/rolebindings/rolebinding.yaml:10]: Looks like the serviceAccount isn't currently deployed in the curator-operator-system namespace, you will have to add it under apps/curator-operator-system/base

Comment (skanthed) [reply curator-operator-system/base/rbac.authorization.k8s.io/rolebindings/rolebinding.yaml:10]: Okay. I was creating a service account while deploying the project.

Comment (skanthed) [reply curator-operator-system/base/rbac.authorization.k8s.io/rolebindings/rolebinding.yaml:10]: Added service account.

File: curator-operator-system/base/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- rbac.authorization.k8s.io/
```
[the remark this answers is not in the document]
Comment (skanthed) [reply curator-operator-system/base/kustomization.yaml:4]: Updated

**Comment by skanthed** [comment]
@dystewart Updated the commit message.

**Pull request merged**